.circleci/config.yml

```diff
@@ -64,7 +81,7 @@ jobs:
                   "type": "section",
                   "text": {
                     "type": "mrkdwn",
-                    "text": " The Tidepool Web UI tests have failed. :sad_tapani: \n\n *Please confirm resolution details:*"
+                    "text": "*:x: Tidepool Web UI Tests Failed* :sad_tapani: \n\n *Summary:* \n *$PASSED_TESTS/$TOTAL_TESTS tests passed* \n *$FAILURES tests failed* \n *$ERRORS errors encountered* \n\n :link: *<${CIRCLE_BUILD_URL}|View CircleCI Build Details>* \n\n :exclamation: *Please confirm resolution details.*"
```

Comment:
Can we remove the resolution details section +checkboxes all together? I originally though the checkboxes would work, but you technically need an api set up to receive the checkbox request and post it back to slack which seems overkill. I'm happy to add anything to make this more useful or we can just commit to commenting on the slack thread when the tests fail.

Reply:
Will do + should I tag someone in the message? You, Brian? Or some group?